src/leap/mail/mail.py

```diff
@@ -190,11 +212,15 @@ def get_subpart(self, part):
 
         sub_pmap = self._pmap.get("part_map", {})
         try:
-            part_map = sub_pmap[str(part + 1)]
+            part_map = sub_pmap[str(part)]
         except KeyError:
-            logger.debug("getSubpart for %s: KeyError" % (part,))
+            log.msg("getSubpart for %s: KeyError" % (part,))
```

Comment:
Also, for the long run I think we should standarize all the twisty-things to use twisted log facility, it behaves betterfor async logging. And I believe there's been lots of improvements in 15.0